### ISSUE TITLE
syncing between devices changes modified date of all notes to current date/time

### ISSUE THREAD

**Issue by kiyv** [standardnotes/web#205]
i also experience this: https://github.com/standardnotes/web/issues/196

**Comment by mobitar** [comment]
Hey @nestiio, are you experiencing two separate issues? Because the title is different from what #196 is. Can you describe the behavior you mentioned in your title a bit more?

**Comment by nestiio** [comment]
Yes, i am experiencing two separate issues. i was just saying i'm experiencing #196 too. Here's an explanation of my title:

what happened..

1. I synced my notes to the standard notes server with the internet using my mobile device
> I use the SN app on mobile (ios) to write notes.

2. I opened my pc (windows) with the wifi off and logged out of the SN desktop app while offline, to make sure no syncing error would happen.

> I haven't logged in on SN on any other device for more than a week

3. I login on the desktop app. the time i sorted the notes to their modified date, all of my notes' modified date are listed as the current date of the time i logged in on desktop.

> All of my notes that current time's modified dates had changed into the current time of sync. their created dates are still normal.

![issue](https://user-images.githubusercontent.com/39004909/40588642-40c101ae-61d0-11e8-85d2-8ab791c55847.png)
